apps/frontend/src/app/components/auth/register.tsx

```diff
@@ -1,6 +1,7 @@
 import { IonButton, IonButtons, IonCardContent, IonCardSubtitle, IonCardTitle, IonContent, IonHeader, IonInput, IonItem, IonMenuButton, IonPage, IonRouterLink, IonTitle, IonToolbar } from '@ionic/react';
```

Comment:
Traditionally React filenames follow their component name. For example: `Register.tsx` and `SignInWithGoogle.tsx`